### PR TITLE
mobile: translations

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 1]
```diff
@@ -160,6 +170,11 @@
         <target>Проверяются новые сообщения...</target>
         <note>notification</note>
       </trans-unit>
+      <trans-unit id="Choose file" xml:space="preserve">
+        <source>Choose file</source>
+        <target>Выберите файл</target>
+        <note>No comment provided by engineer.</note>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:176]:
```suggestion
        <target>Выбрать файл</target>
        <note>Attachment option to choose file.</note>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 2]
```diff
@@ -285,6 +300,11 @@
         <target>Создать профиль</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="Currently maximum supported file size is %@." xml:space="preserve">
+        <source>Currently maximum supported file size is %@.</source>
+        <target>Максимальный размер файла - %@.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:305]: the translation goes as following in android, maybe we should change it there as well
```suggestion
        <target>В настоящее время максимальный поддерживаемый размер файла составляет %@.</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 3]
```diff
@@ -395,6 +415,11 @@
         <target>Ошибка: неверная ссылка</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="File will be received when your contact is online, please wait or check later!" xml:space="preserve">
+        <source>File will be received when your contact is online, please wait or check later!</source>
+        <target>Файл будет принят, когда ваш контакт онлайн, подождите или проверьте позже!</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:420]:
```suggestion
        <target>Файл будет принят, когда ваш контакт будет в сети, подождите или проверьте позже!</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 4]
```diff
@@ -440,6 +465,26 @@
...
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="Ignore" xml:space="preserve">
+        <source>Ignore</source>
+        <target>Не отвечать</target>
+        <note>ignore incoming call via notification</note>
+      </trans-unit>
+      <trans-unit id="Image will be received when your contact is online, please wait or check later!" xml:space="preserve">
+        <source>Image will be received when your contact is online, please wait or check later!</source>
+        <target>Картинка будет принята, когда ваш контакт онлайн, подождите или проверьте позже!</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:475]:
```suggestion
        <target>Картинка будет принята, когда ваш контакт будет в сети, подождите или проверьте позже!</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 5]
```diff
@@ -700,6 +750,16 @@ To connect, please ask your contact to create another connection link and check
...
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="Waiting for file" xml:space="preserve">
+        <source>Waiting for file</source>
+        <target>Ожидается прием файла</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Waiting for image" xml:space="preserve">
+        <source>Waiting for image</source>
+        <target>Ожидается прием картинки</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:760]:
```suggestion
        <target>Ожидается прием изображения</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 6]
```diff
@@ -802,6 +862,11 @@ You can cancel this connection and remove the contact (and try later with a new
 Вы можете отменить соединение и удалить контакт (и попробовать позже с другой ссылкой).</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="Your contact sent a file that is larger than currently supported maximum size (%@)." xml:space="preserve">
+        <source>Your contact sent a file that is larger than currently supported maximum size (%@).</source>
+        <target>Ваш контакт послал файл больше максимально разрешенного размера (%@).</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:867]: same - maybe should change in android
```suggestion
        <target>Ваш контакт отправил файл, размер которого превышает поддерживаемый в настоящее время максимальный размер (%@).</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 7]
```diff
@@ -859,10 +934,15 @@ SimpleX серверы не могут получить доступ к ваше
         <target>соединитесь с разработчиками.</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="connected" xml:space="preserve">
+        <source>connected</source>
+        <target>соединено</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:939]:
```suggestion
        <target>соединение установлено</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 8]
```diff
@@ -952,10 +1087,150 @@ SimpleX серверы не могут получить доступ к ваше
         <note>Bundle name</note>
       </trans-unit>
       <trans-unit id="NSCameraUsageDescription" xml:space="preserve">
-        <source>SimpleX needs camera access to scan QR codes to connect to other app users</source>
+        <source>SimpleX needs camera access to scan QR codes to connect to other users and for video calls.</source>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:1090]: add same for translation?

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 9]
```diff
@@ -952,10 +1087,150 @@ SimpleX серверы не могут получить доступ к ваше
...
+    </body>
+  </file>
+  <file original="SimpleX NSE/en.lproj/Localizable.strings" datatype="plaintext" source-language="en" target-language="ru">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="13.3" build-num="13E113"/>
+    </header>
+    <body>
+      <trans-unit id="%@ is connected!" xml:space="preserve">
+        <source>%@ is connected!</source>
+        <target>%@ присоединен!</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:1135]: "присоединен" sounds very strange in this context to me
```suggestion
        <target>соединение с %@ установлено!</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 10]
```diff
@@ -952,10 +1087,150 @@ SimpleX серверы не могут получить доступ к ваше
...
+   chat list item title</note>
+      </trans-unit>
+      <trans-unit id="connection established" xml:space="preserve">
+        <source>connection established</source>
+        <target>соединение установлено</target>
+        <note>chat list item title (it should not be shown</note>
+      </trans-unit>
+      <trans-unit id="connection:%@" xml:space="preserve">
+        <source>connection:%@</source>
+        <target>connection:%@</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:1181]:
```suggestion
        <target>соединение:%@</target>
```

Comment (epoberezkin) [reply apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:1181]: not in the ui

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff (hunk 2 above)
Comment (epoberezkin) [reply apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:305]: I don't think it's needed...

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff (hunk 8 above)
Comment (epoberezkin) [reply apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:1090]: done

**Pull request merged**